### PR TITLE
Basic website

### PR DESCRIPTION
**Pull request by marceloyuela**
The basic (working) website in its own project. This is nowhere near the final site.

JSON functionality is next.

### CODE REVIEW

**Comment by ILMTitan** [comment]
LGTM

File: Fix-Website/ApplicationInsights.config [hunk 1]
```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (chrsmith) [inline Fix-Website/ApplicationInsights.config]: What is this ApplicationInsights file? I assume it is not necessary?

File: Fix-Website/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
```
Comment (chrsmith) [inline Fix-Website/Properties/AssemblyInfo.cs]: Is this file even used? If so, we can safely delete it.

File: Fix-Website/Stylesheets/toc.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.TOC {
```
Comment (chrsmith) [inline Fix-Website/Stylesheets/toc.css]: Please review the Google style guide for HTML and CSS.
https://google.github.io/styleguide/htmlcssguide.xml

- Use a namespace, e.g. `.gtp-toc, .gtp-info`, etc.
- All lower case.

File: Fix-Website/Templates/TOC.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (chrsmith) [inline Fix-Website/Templates/TOC.html]: I'd delete this unnecessary vertical whitespace.

File: Fix-Website/Templates/content-table.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿<section class="TOC">
```
Comment (chrsmith) [inline Fix-Website/Templates/content-table.html]: Please use the `.ng` suffix for HTML partials. Why .ng? I have no idea. But it does make it clear that it isn't a complete web page, and only a partial.

File: Fix-Website/Templates/info.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    <div ng-show="checkFrame(2)">
+        <h1>Google Cloud Platform DNS</h1>
+        <table>
+            <tr>
+                <td>Get-GcdChange</td>
+                <td>Lists the ManagedZone's Change resources.</td>
+            </tr>
+        </table>
+    </div>
+    <div ng-show="checkFrame(3)">
```
Comment (chrsmith) [inline Fix-Website/Templates/info.html]: Where is `checkFrame` defined? Try to use an explicit controller reference, as it makes it clearer what is going on. e.g.

````
<div ng-controller="WidgetCtrl as widgetCtrl">
  <h1>{{ widgetCtrl.title }}</h1>
</div>
````

File: Fix-Website/app/app.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
+(function() {
+    var app = angular.module('powershellSite', []);
+
+    app.directive("contentTable", function(){
+        return {
+            restrict: 'E',
+            templateUrl: "Templates/content-table.html",
+            controller: function($scope) {
```
Comment (chrsmith) [inline Fix-Website/app/app.js]: Please tease out the ctonroller into a separate declaration. e.g.

````
/**
 * Controller for doing stuff.
 */
var widgetController = function($scope) {
  ...
};

...

controller: widgetController
````

**Comment by chrsmith** [comment]
Let some comments. Generally the CSS and JavaScript is pretty far from the Google style guide. We can ignore some things since we aren't using Closure or google3. But we should strive to be as style-compliant as possible.

Feel free to just make these changes in a future PR.

File: Fix-Website/ApplicationInsights.config (hunk 1 above)
Comment (marceloyuela) [inline Fix-Website/ApplicationInsights.config]: Some of these files are fragments (I assume) from visual studio's basic website project config. Will clean up.

**Comment by marceloyuela** [comment]
Yeah for sure. Some more stuff beyond what you posted isn't in accordance to style for google websites (See: font). I'll be sure to spend a long time on fixing these and (other) issues that come from creating a website that fits google standards. Most of these fixes will be in the next PR, along with (rough) JSON reading capabilities.

**Pull request merged**